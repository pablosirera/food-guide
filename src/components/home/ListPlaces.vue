<script setup>
import ImageFullCard from '@/components/ui/ImageFullCard.vue'

defineProps({
  places: {
    type: Array,
    default: () => [],
  },
})
</script>

<template>
  <section class="flex flex-wrap justify-between sm:justify-center">
    <ImageFullCard
      v-for="(place, index) in places"
      :key="index"
      :item="place"
    />
  </section>
</template>
