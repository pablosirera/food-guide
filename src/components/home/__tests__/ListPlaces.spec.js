import { mount } from '@vue/test-utils'
import ListPlaces from '@/components/home/ListPlaces.vue'
import { describe, expect, it } from 'vitest'

describe('ListPlaces.spec.js', () => {
  it('renders', () => {
    const wrapper = mount(ListPlaces)
    expect(wrapper.exists).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
