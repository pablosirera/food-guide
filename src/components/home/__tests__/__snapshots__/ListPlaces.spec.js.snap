// Vitest Snapshot v1

exports[`ListPlaces.spec.js > renders 1`] = `"<section class=\\"flex flex-wrap justify-between sm:justify-center\\"></section>"`;
